noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/aeffacer/ArrayCoordinateListVisitor.java

```diff
@@ -31,7 +31,7 @@
  * or contact directly:
  * info_at_ orbisgis.org
  */
-package org.noise_planet.noisemodelling.pathfinder;
+package org.noise_planet.noisemodelling.pathfinder.aeffacer;
```

Comment:
what is aeffacer ?

Reply:
it's all the classes we didn't use in NM,
so Pierre told me to group them in a package instead of deleting them.

Reply:
@pierromond why do you want to keep unused class ?